frontend/src/pages/modelServing/screens/projects/ModelServingPlatform.tsx

```diff
@@ -92,6 +92,43 @@ const ModelServingPlatform: React.FC = () => {
     }
   };
 
+  const renderPlatformEmptyState = () => {
+    if (kServeEnabled || modelMeshEnabled) {
+      return (
+        <EmptyDetailsList
+          title={modelMeshEnabled ? 'Start by adding a model server' : 'Start by deploying a model'}
+          description={
+            modelMeshEnabled
+              ? 'Model servers are used to deploy models and to allow apps to send requests to your models. Configuring a model server includes specifying the number of replicas being deployed, the server size, the token authorization, the serving runtime, and how the project that the model server belongs to is accessed.'
+              : 'Each model is deployed on its own model server.'
+          }
+          actions={
+            <ModelServingPlatformButtonAction
+              isProjectModelMesh={isProjectModelMesh}
+              testId={`${isProjectModelMesh ? 'add-server' : 'deploy'}-button`}
+              emptyTemplates={emptyTemplates}
+              onClick={() => {
+                setPlatformSelected(
+                  isProjectModelMesh ? ServingRuntimePlatform.MULTI : ServingRuntimePlatform.SINGLE,
+                );
+              }}
+              key={`${isProjectModelMesh ? 'multi' : 'single'}-serving-actions`}
+            />
```

Comment:
Showing the button here causes two buttons on the screen for `Deploy model`. 

![image](https://github.com/opendatahub-io/odh-dashboard/assets/11633780/41ee780e-f662-417c-a7b3-862bba396e9e)

We should not add it to the toolbar in this case.